src/target/llvm/codegen_llvm.cc

```diff
@@ -698,21 +698,33 @@ llvm::Value* CodeGenLLVM::CreateIntrinsic(const CallNode* op) {
         sig_type.push_back(arg_value.back()->getType());
       }
     }
+    // LLVM's prefetch intrinsic returns "void", while TVM's prefetch
+    // returns int32. This causes problems because prefetch is one of
+    // those intrinsics that is generated automatically via the
```

Comment:
I see, let us leave a TODO here, hopefully after we introduce the type system to type the intrinsics, we will be able to use the same type as LLVM

Reply:
Will do.